apps/andi/lib/andi/input_schemas/organizations.ex

```diff
@@ -91,9 +105,17 @@ defmodule Andi.InputSchemas.Organizations do
 
   def is_unique?(id, org_name) do
     from(org in Andi.InputSchemas.Organization,
-      where: org.orgName == ^org_name and org.id != ^id
+      where: org.orgName == ^org_name or org.id == ^id
     )
     |> Repo.all()
     |> Enum.empty?()
   end
+
+  def org_title_to_org_name(org_title) do
```

Comment:
it might be nice to DRY this up a bit because the datasets module does the same thing